src/StorageStack.ts

```diff
@@ -330,4 +332,49 @@ export class StorageStack extends Stack {
     return user;
   }
 
+  setupManagementUser(bucketArns: string[], sseKeyArn: string) {
+    const user = new iam.User(this, 'management-user');
+    const key = new iam.AccessKey(this, 'management-user-key', {
+      user: user,
+    });
+    new Secret(this, 'management-user-secret', {
+      secretStringValue: key.secretAccessKey,
+    });
+    user.addToPolicy(new iam.PolicyStatement({
+      sid: 'AllowToListTheBucketsInTheAccount',
+      effect: iam.Effect.ALLOW,
+      actions: ['s3:ListAllMyBuckets'],
+      resources: ['*'],
+    }));
+    user.addToPolicy(new PolicyStatement({
+      actions: [
+        'kms:Decrypt',
+        'kms:DescribeKey',
+        'kms:Encrypt',
+        'kms:ReEncrypt*',
+        'kms:GenerateDataKey*',
+        'kms:Decrypt',
+      ],
+      effect: Effect.ALLOW,
+      resources: [sseKeyArn],
+    }));
+
+    user.addToPolicy(new PolicyStatement({
+      actions: [
+        's3:PutObject',
+        's3:PutObjectLegalHold',
+        's3:PutObjectRetention',
+        's3:PutObjectTagging',
+        's3:PutObjectVersionTagging',
+        's3:Abort*',
+        's3:GetObject*',
+        's3:GetBucket*',
+        's3:List*',
```

Comment:
Waar komt deze specifieke lijst vandaan? Is dit wat normaal in grantreadwrite wordt toegevoegd?

Reply:
Dit heb ik uit de policy van bucket.grandRead en bucket.grantPut (geen delete voor deze user)
Die functies voegen deze acties op de Key en Bucket toe. Ik kan deze functies niet gebruiken om dat de policy te groot wordt voor cloudformation